packages/modules/vehicles/smarteq/api.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+
+import logging
+from typing import Union
+import asyncio
+import json
+from modules.common.store import RAMDISK_PATH
+from modules.vehicles.smarteq.config import SmartEQ
+import requests
+import bs4
+import pkce
+import pickle
+
+date_fmt = '%Y-%m-%d %H:%M:%S'
+# refreshToken_exp_days = 7    # 7 days before refreshToken expires a new refreshToken shall be stored
+initialToken = '1.2.3'
+
+log = logging.getLogger("soc."+__name__)
+
+# Constants
+bASE_URL = "https://id.mercedes-benz.com"
```

Comment:
```suggestion
BASE_URL = "https://id.mercedes-benz.com"
```
Die Schreibweise mit den Großbuchstaben war richtig, da es Konstanten sind. Diese werden in Python mit GROSSBUCHSTABEN benannt.